### DISCUSSION TITLE
Cloud storage in the file explorer

### DISCUSSION THREAD

**Discussion by fdekerme** [standardnotes/forum#3820] [Suggestions]
Hello,
Would it be possible to make a "Standard Notes" folder appear in the file explorer to generate the synchronized files in SN? I have the "Pro" subscription with 100GB encrypted cloud storage, and would like to be able to use it more flexibly than just via the app. In particular, it would allow me to modify the files attached to a note without having to re-upload them afterwards, by saving them directly in the SN cloud.
Thanks

**Comment by r0-n** [comment]
I wanted to suggest something exactly like this.
Please consider making the storage more accessible like a cloud drive.
Managing files/folders currently is (and feels) cumbersome.

**Reply by fdekerme** [reply]
I had also proposed on Discord (many more active than here) but apparently it's "out of scope"
https://discord.com/channels/894600255058898974/1317966140055748638/1317966140055748638
